client/app/services/HelpTrigger.jsx

```diff
@@ -1,9 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import cx from 'classnames';
```

Comment:
Why `cx`?

Reply:
I don't know actually but it's what we used in FB and also what everyone else uses - including the official npm library. I like it cause it's short.

Reply:
👍